doc/assets/js/api-list.js

```diff
@@ -97,15 +98,17 @@ tabview.get('panelNode').all('a').each(function (link) {
 
 // -- Private Functions --------------------------------------------------------
 function getFilterResultNode() {
-    return filter.get('queryType') === 'classes' ? classesNode : modulesNode;
+    var queryType = filter.get('queryType');
```

Comment:
This is third-party (YUIDoc) code that, for better or worse, is tracked at the moment by Git.  But I do prefer your version!